src/target/riscv/riscv-013.c

```diff
@@ -1385,6 +1385,11 @@ static int fpr_read_progbuf(struct target *target, uint64_t *value,
 {
 	assert(target->state == TARGET_HALTED);
 	assert(number >= GDB_REGNO_FPR0 && number <= GDB_REGNO_FPR31);
+	if (!has_sufficient_progbuf(target, 2)) {
+		LOG_DEBUG("Skipping FPR read: insufficient progbuf (size=%d)",
```

Comment:
[here and around] Please, use `LOG_TARGET_DEBUG(...)` to give a bit more context:
```suggestion
		LOG_TARGET_DEBUG(target, "Skipping FPR read: insufficient progbuf (size=%d)",
```

Reply:
Changed it in the updated version.